### ISSUE TITLE
[production][users#toggle_confirmation] undefined method `confirm' for #<User:0x0000000005f6d2b8>
Did you mean?  confirmed?

### ISSUE THREAD

**Issue by Ana06** [openSUSE/osem#2498]
users#toggle_confirmation threw this 0 times
<pre>
undefined method `confirm&#39; for #&lt;User:0x0000000005f6d2b8&gt;
Did you mean?  confirmed?
</pre>
[https://events.opensuse.org/admin/users/12016/toggle_confirmation?user[to_confirm]=true](https://events.opensuse.org/admin/users/12016/toggle_confirmation?user[to_confirm]=true)"

[See this exception on Errbit](https://errbit-opensuse.herokuapp.com/apps/5620ca99dc71fa00b1000002/problems/5cc2e9f6b0f77c000814938a)

**Comment by differentreality** [comment]
Hmmm you cannot confirm a user if you are using ICHAIN - would it make any sense to do so? Why do people want to confirm users while using ICHAIN?

This is caused by the lack of confirmable in Devise when using ICHAIN https://github.com/openSUSE/osem/blob/master/app/models/user.rb#L48

**Comment by Ana06** [comment]
@differentreality 

> Hmmm you cannot confirm a user if you are using ICHAIN - would it make any sense to do so? Why do people want to confirm users while using ICHAIN?

You are right, no. But there are already confirmed and unconfirmed users and this raises an exception.

**Comment by differentreality** [comment]
The confirmed users exist since before ICHAIN was implemented in OSEM, when users actually created their accounts in OSEM, as such OSEM would confirm their email address.

So what are we expecting to be done for this? Not showing the confirm button in the admin panel?

**Comment by Ana06** [comment]
@differentreality 

> So what are we expecting to be done for this? Not showing the confirm button in the admin panel?

Not showing the button and that all the users are confirmed. We have some unconfirmed users in events.opensuse.org
